cmake/modules/extensions.cmake

```diff
@@ -5280,6 +5302,89 @@ function(add_llext_target target_name)
   set_target_properties(${target_name} PROPERTIES
     lib_target ${llext_lib_target}
     lib_output ${llext_lib_output}
+    pkg_input  ${llext_pkg_input}
     pkg_output ${llext_pkg_output}
   )
 endfunction()
+
+# Usage:
+#   add_llext_command(
+#     TARGET <target_name>
+#     PRE_BUILD | POST_BUILD | POST_PKG
+#     COMMAND <command> [...]
+#   )
+#
+# Add a custom command to an llext target that will be executed during
+# the build. The command will be executed at the specified build step and
+# can refer to <target>'s properties for build-specific details.
+#
+# The differrent build steps are:
```

Comment:
Minor typo in comment. Obviously not blocking. Do not force-push and lose approvals ONLY for this.